js/styles/_cookie-banner.scss

```diff
@@ -15,7 +15,11 @@ $border-bottom-width: govuk-spacing(2);
   // when user changes colours in their browser.
   border-bottom: $border-bottom-width solid transparent;
 
-  background-color: $govuk-canvas-background-colour;
+  @include _govuk-rebrand(
+    "background-color",
+    $govuk-template-background-colour,
+    $_govuk-rebrand-template-background-colour
+  );
```

Comment:
this is basically the whole change, only the background colour is updated if cookie banner is displayed on a rebranded page